src/components/container/toDoList/toDo.js

```diff
@@ -1,24 +1,26 @@
-import React, { Component, PropTypes } from 'react'
-import { connect } from 'react-redux'
-import { deleteToDo } from '../../../actionCreators/toDoList_action_creator'
-import { toggleToDo } from '../../../actionCreators/toDo_action_creator'
-import ToDo from '../../presentation/toDoList/toDo'
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { deleteToDo } from '../../../actionCreators/toDoList_action_creator';
+import { toggleToDo } from '../../../actionCreators/toDo_action_creator';
+import ToDo from '../../presentation/toDoList/toDo';
 
 class ToDoContainer extends Component{
   _onToggle = () => {
+    //Add toggle to propTypes
```

Comment:
So even though its being passed down via connect in this file its still worth adding to propTypes?